src/components/UserAccount/index.jsx

```diff
@@ -115,9 +129,50 @@ class UserAccount extends React.Component {
               </Row>
             </Col>
             <Col md={8} lg={9} className="pt-md-1">
-
-              <h3>About {firstName}</h3>
-              <p>{aboutMe}</p>
+              <EditableContent
+                fieldName="about"
+                isEditing={this.state.currentlyEditingField === 'about'}
+                disabled={false}
+                renderStatic={props => ( // eslint-disable-line no-unused-vars
```

Comment:
@douglashall Is this a crazy way to do editable fields? The EditableContent component takes 3 render props: renderStatic, renderEditable, renderEditing.